linkid_to_patid.py

```diff
@@ -157,16 +166,28 @@ def translate_linkids(args):
             source_name=source_metadata_filename,
             linkage_name=args.hhlinkszip,
         )
-        if len(metadata_issues) == 0:
-            write_hh_links(args)
+        if not args.force:
+            if len(metadata_issues) == 0:
+                write_hh_links(args)
+                return
+            else:
+                print(
+                    f"ERROR: Inconsistencies found in source "
+                    f"metadata file {args.sourcefile}"
+                    f" and linkage archive metadata in {args.linkszip}:"
+                )
+                for issue in metadata_issues:
+                    print("\t" + issue)
+                return
         else:
-            print(
-                f"ERROR: Inconsistencies found in source "
-                f"metadata file {args.sourcefile}"
-                f" and linkage archive metadata in {args.linkszip}:"
-            )
-            for issue in metadata_issues:
-                print("\t" + issue)
+            if len(metadata_issues) > 0:
+                print(
+                    f"WARNING: Inconsistencies found in source "
+                    f"metadata file {args.sourcefile}"
+                    f" and linkage archive metadata in {args.linkszip}."
+                    f"Proceeding anyway (this may yield meaningless data)"
+                )
```

Comment:
I would suggest we still print out the issues here anyway, even if skipping them with the -f flag. There may be opportunity to rewrite this `if` with a little less duplication if we do that too. Maybe something like

```python
        if len(metadata_issues) > 0:
            print(
                f"{"ERROR" if args.force else "WARNING"}: "
                f"Inconsistencies found in source "
                f"metadata file {args.sourcefile}"
                f" and linkage archive metadata in {args.linkszip}:"
            )
            for issue in metadata_issues:
                print("\t" + issue)

        if len(metadata_issues) == 0 or args.force:
            write_hh_links(args)
```

Reply:
The above changes have been adopted